lib/mix/tasks/skf.gen.makefile.ex

```diff
@@ -2,7 +2,7 @@ defmodule Mix.Tasks.Skf.Gen.Makefile do
   use Mix.Task
 
   @template Path.join(["skf.gen.makefile", "Makefile"])
-  @switches [cloud: :string]
+  @switches [docker: :boolean, cloud: :string]
```

Comment:
I'm not sure if it is planned to be added in later phase, however it would be nice to have somewhere visible which values does `cloud` accept  without having to search the code.

Reply:
I also agree that some help would be useful, e.g. when the mix task is invoked without args. Not necessarily a part of this pull, but it would be good to have that in the future.